exercise_051_cluster_singleton_akka_bootstrap_dns/src/main/resources/application.conf

```diff
@@ -0,0 +1,72 @@
+## Recommended settings for robust cluster
+akka.cluster.shutdown-after-unsuccessful-join-seed-nodes = 30s
+akka.coordinated-shutdown.exit-jvm = on
+
+akka {
+
+  io.dns.resolver = async-dns
+
+  log-config-on-start = on
+  discovery.method = akka-dns
+  management.cluster.bootstrap.contact-point-discovery {
+    service-name = "_picluster._tcp.lb.com"
```

Comment:
If I'm reading https://doc.akka.io/docs/akka/current/discovery/index.html#discovery-method-dns right, shouldn't this be `service-name = "picluster.lb.com` to get SRV requests for `_8558._tcp_picluster.lb.com` (or `_management._tcp._picluster.lb.com`)?

Reply:
Indeed: now that I get the mechanism, it all makes sense.
Will adapt accordingly.
Also, verified that this now works.